col/src/main/java/vct/col/ast/print/AbstractPrinter.java

```diff
@@ -55,18 +55,12 @@ public void nextExpr(){
       expr_level=0;
     }
   }
-  
-  private static final Origin missing=new MessageOrigin("unknown location");
-  
+
   public void pre_visit(ASTNode node){
     super.pre_visit(node);
     if (in_expr) {
       expr_level++;
     }
-    Origin o=node.getOrigin();
```

Comment:
Is it not possible to get a `none` value here? or how is this managed in case we get one? it will just print `none`?

Reply:
I am not sure what you mean. The reason I removed this code is because o is never used even if it is assigned. I might have been a programming error however this pull request is not to fix bugs but to remove unused code. I did make an issue for it. https://github.com/utwente-fmt/vercors/issues/675 However I do not think it is worthwhile to investigate it because it is going to be rewriten.

Reply:
I am not sure what you mean. The reason I removed this code is because o is never used even if it is assigned. I might have been a programming error however this pull request is not to fix bugs but to remove unused code. I did make an issue for it. #675 However I do not think it is worthwhile to investigate it because it is going to be rewriten.